dataprep/clean/clean_phone.py

```diff
@@ -265,12 +295,20 @@ def _check_phone(phone: Any, clean: bool) -> Any:
         return (None,) * 5 + ("unknown",) if clean else False
     if mch.group("country") and not mch.group("area"):
         return (None,) * 5 + ("unknown",) if clean else False
+    if mch.group("letters"):
+        # Check that there are 7 alphanumeric characters present
+        letters = re.sub(r"\W+", "", mch.group("letters"))
+        if len(letters) != 7:
+            return (None,) * 5 + ("unknown",) if clean else False
+        # Convert letters to numbers
+        numlist = [ALPHA_NUM_MAP[char] if char.isalpha() else char for char in letters]
+        numbers = "".join(str(num) for num in numlist)
```

Comment:
Can you give an example of when `numlist` contains a type other than string? I'm wondering why we can't just have `"".join(numlist)` here.

Reply:
Originally the letters were mapped to integers, but then I changed them to strings. So yes, `str(num)` is no longer required here.